src/pages-and-resources/discussions/app-list/AppList.scss

```
@@ -6,7 +6,8 @@
   .pgn__data-table-container {
     tr {
       th {
-        background-color: $white;
+        background-color: white;
+        background-color: var(--white);
```

Has this been declared twice for compatibilty with old/odd browsers ?
If its to have a fallback default in case of unavailability of the variable, then we could write it something like :
```suggestion
        background-color: var(--white, white);
```

---

It's for compatibility.